packages/cli/src/tools/config/__tests__/__snapshots__/index-test.js.snap

```diff
@@ -1,5 +1,60 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`should automatically put "react-native" into haste config 1`] = `
+Object {
+  "platforms": Array [
+    "ios",
+    "android",
+  ],
+  "providesModuleNodeModules": Array [
+    "react-native",
+  ],
+}
+`;
+
+exports[`should have a valid structure by default 1`] = `
+Object {
+  "assets": Array [],
+  "commands": Array [],
+  "dependencies": Object {},
+  "haste": Object {
+    "platforms": Array [],
+    "providesModuleNodeModules": Array [],
+  },
+  "platforms": Object {},
+  "project": Object {},
+  "reactNativePath": ".",
+  "root": "<<REPLACED>>",
+}
+`;
+
+exports[`should load an out-of-tree "windows" platform that ships with a dependency 1`] = `
+Object {
+  "haste": Object {
+    "platforms": Array [
+      "windows",
+    ],
+    "providesModuleNodeModules": Array [
+      "react-native-windows",
+    ],
+  },
+  "platforms": Object {
+    "windows": Object {},
+  },
+}
+`;
+
+exports[`should load commands from "react-native-foo" and "react-native-bar" packages 1`] = `
+Array [
+  Object {
+    "name": "foo-command",
+  },
```

Comment:
Can we leave one command as is and add `func` to the other one? This way we can have some logic tested for commands that are valid and the ones that are not (we could also inspect stdout to check if there's a warning message)

Reply:
I think it's a good idea, how about we do it in a follow-up PR where I do the "skip invalid packages" feature? That way, I can test that "package with invalid command is not available in the output" and have both capabilities tested at the same time (check resulting object and inspect stdout)

Reply:
Hm, lines https://github.com/react-native-community/cli/pull/295/files#diff-60a96ddc584a7d7c65942648e9e88c57R158 and https://github.com/react-native-community/cli/pull/295/files#diff-60a96ddc584a7d7c65942648e9e88c57R149 indicate that `func` should be there. Can you investigate?

Reply:
I run the test with `-u` flag. Maybe `jest` is not keeping functions in a snapshot?

Reply:
it should show a `[Function]` replacement, so rather nope

Reply:
Logging the config right before returning from the function:
```
[ { name: 'foo-command', func: [Function: func] },
        { name: 'bar-command', func: [Function: func] } ]
```
the properties are there.

Reply:
I need to run to a meeting now, would appreciate if you could investigate it further.

Reply:
I'd blame `removeString` then

Reply:
Could be

Reply:
I can actually remove it from here.